src/request-handler.js

```diff
@@ -82,18 +82,23 @@ export class RequestHandler {
       const log = helper.getLog();
       let res;
       const { method, url } = request;
+      const headers = Object.fromEntries(request.headers);
+      const forwardedContext = {
+        headers,
+        ...context,
+      };
       const loggableRequest = {
         method,
         url,
-        headers: Object.fromEntries(request.headers),
+        headers,
```

Comment:
Would you want to log all of the headers? What about Authorization?

Reply:
I just kept the functionality on that line the same as it was before.